src/einsteinpy/symbolic/tensor.py

```diff
@@ -33,6 +34,24 @@ def __init__(self, arr):
     def __getitem__(self, index):
         return self.arr[index]
 
+    def __str__(self):
+        """
+        Returns a String with a readable representation of the object of class Tensor
+
+        """
+        representation = "Tensor" + "\n"
+        representation += self.arr.__str__()
+        return representation
+
+    def __repr__(self):
+        """
+        Returns a String with a representation of the state of the object of class Tensor
+
+       """
+        interpretable_representation = self.__class__.__name__
+        interpretable_representation += self.arr.__str__()
```

Comment:
Call `self.arr.__repr__()` here.

Also it would be great if you could show some screenshots of the output.
I think, you didn't quite understand the utility of `__str__` and `__repr__`, I may be wrong.

Reply:
![image](https://user-images.githubusercontent.com/51050291/60673783-dea30300-9e78-11e9-8283-b2003d7263f3.png)
This is the result with __repr__
In fact I had  put __str__ here as in the other. something that I couldn't understood why. So now that you said me to change it, make sense again 

Correct me if I am wrong, but the method __str__ is used to print human readable information about the object and __repr__ if to give more technical information about it right?